metabase/metabase
# series breakout for scatter plot

Closes https://github.com/metabase/metabase/issues/35802

### Description

Handles x-axis series breakouts for the echarts' static scatter plot.

Implementation is in `model.ts`, rest is adding a storybook example.

### How to verify

1. Create a scatter plot with an x-axis breakout
2. Add to a dashboard, send a subscription
3. Confirm the chart with the breakout renders correctly

### Demo

![Screenshot 2023-12-14 at 2.35.31 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/0bab1677-0835-4d6a-bfd9-a4907032df6a.png)


### Checklist
